### PR TITLE
Fix: Add missing xlarge to vanilla theme

### LINKED ISSUE #431: Extra breakpoints required. Missing xlarge.
Following the breakpoint update in core, we need to add in an extra large node for headers and backgrounds.

https://github.com/adaptlearning/adapt-contrib-vanilla/blob/aac3a2bdade9398bc9cf353dd40b64f5235096dd/js/themePageView.js#L46-L55

linked to issue #430 

We could add this issue in now and retain the existing functionality. See issue #430 as functionality may get updated at a later date in this issue.

### PR DESCRIPTION
**Pull request by StuartNicholls**
Added xlarge to theme. 

Fixes #431

### CODE REVIEW

**Comment by guywillis** [comment]
Looking great. Will approve after the files have been run through with ESLint to remove erroneous white space.

File: example.json [hunk 1]
```diff
@@ -32,7 +34,8 @@
             "_instruction": ""
         },
         "_backgroundImage": {
-            "_large": "",
+        	"_xlarge": "",
```
Comment (oliverfoster) [inline example.json:37]: Tab insertion, use spaces.

**Review by oliverfoster — APPROVED** [review]

**Review by guywillis — APPROVED** [review]
👍

File: js/themeView.js [hunk 2]
```diff
@@ -57,70 +55,30 @@ export default class ThemeView extends Backbone.View {
...
-    if (backgroundImage) {
-      this.$el.addClass('has-bg-image');
-      this.$background
-        .css('background-image', 'url(' + backgroundImage + ')');
-      return;
-    }
-
-    this.$el.removeClass('has-bg-image');
-    this.$background.css('background-image', '');
+    const backgroundImage = backgroundImages[`_${Adapt.device.screenSize}`] ?? backgroundImages._small;
```
Comment (cahirodoherty-learningpool) [inline js/themeView.js:58]: Why have we moved away from the device singleton?

Comment (StuartNicholls) [reply js/themeView.js:58]: @cahirodoherty-learningpool 

We've already done this in boxmenu

https://github.com/adaptlearning/adapt-contrib-boxMenu/blob/040166743e2a765a7d08bd5b4e6b08a7705ecf64/js/BoxMenuView.js#L73

Not sure, bit less hard-codey / neater..?

Comment (oliverfoster) [reply js/themeView.js:58]: Adapt.device > device.

**Review by ethan-lp — APPROVED** [review]

**Review by kirsty-hames — APPROVED** [review]

**Pull request merged**